less/bootstrap.less

```diff
@@ -43,6 +43,7 @@
 @import "close.less";
 @import "loaders.less";
 @import "tiles.less";
+@import "amount-currency-select";
```

Comment:
For consistency, I think it'd be nicer to include the extension.

Reply:
yup, just noticed myself too :D